Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -26,16 +26,30 @@ public abstract class PaletteBase : Component
         private Padding? _inputControlPadding;
         private PaletteDragFeedback _dragFeedback;
         private string? _themeName;
-        private Font? _baseFont;
 
-        #endregion
+        #region Font Stuff
 
-        #region Public
+        private Font? _baseFont;
```

Comment:
`baseFont` does not need to be nullable, as it is never access outside this class, and is set on first use of Get accesser

Reply:
Removed flag